### ISSUE TITLE
unable to download files from inside a file group

### ISSUE THREAD

**Issue by matthewfischer** [pivotal-cf/pivnet-cli#22]
Given a release that has files in a file-group, I am unable to figure out how to download a file from it. For example, pivotal-container-service, I can list the files available:

```
ubuntu@bosh-stemcell:~$ pivnet product-files -p  pivotal-container-service -r 1.0.4
+--------+-------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------+
|   ID   |                      NAME                       | FILE VERSION  |                                        AWS OBJECT KEY                                         |
+--------+-------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------+
|  76620 | PCF-PKS-NCP OSL                                 |           1.0 | product-files/pivotal-container-service/open_source_licenses-NSX_Container_Plugin-2.1.0.1.txt |
|  76616 | PCF-PKS 1.0 OSL                                 |           1.0 | product-files/pivotal-container-service/open_source_license_PCF-PKS_-_1.0_-_GA.txt            |
| 143263 | pivotal-container-service-1.0.4-build.5.pivotal | 1.0.4-build.5 | product-files/pivotal-container-service/pivotal-container-service-1.0.4-build.5.pivotal       |
+--------+-------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------+
```

The files in the file-groups do not show up, but are visible with another command, the fgs command:

```ubuntu@bosh-stemcell:~$ pivnet fgs -p pivotal-container-service -r 1.0.4
+-----+-----------------------+--------------------------------+
| ID  |         NAME          |       PRODUCT FILE NAMES       |
+-----+-----------------------+--------------------------------+
| 977 | PKS CLI - v1.0.4      | PKS CLI - Mac, PKS CLI -       |
|     |                       | Linux, PKS CLI - Windows       |
| 978 | Kubectl CLIs - v1.9.7 | Kubectl v1.9.7 - Mac, Kubectl  |
|     |                       | v1.9.7 - Linux, Kubectl v1.9.7 |
|     |                       | - Windows                      |
+-----+-----------------------+--------------------------------+
```

But how can I download them? Trying the dlpf command with any glob doesn't seem to see any of the files in the file-group.

**Comment by 4n3w** [comment]
FWIW I was able to download the individual product file with wget, after getting the product-file uri from `pivnet file-groups -p pivotal-container-service -r 1.0.4 --format json | jq .` 

with
  
```wget -d --header="Authorization: Token XXXXXXXXXXXXXXXXXXXXXXX" https://network.pivotal.io/api/v2/products/pivotal-container-service/releases/104377/product_files/143260/download```
(for example)

It would make a lot of sense to be able to download the individual `product-file`, as the entities are readily available in the json output.

**Comment by vitakras** [comment]
We added this feature enhancement to our internal backlog.

**Comment by pivotal-ivan-wang** [comment]
should be fixed in v0.0.51
